src/mono/System.Private.CoreLib/src/ILLink/ILLink.Substitutions.Intrinsics.Vectors.xml

```diff
@@ -0,0 +1,10 @@
+<linker>
+  <assembly fullname="System.Private.CoreLib">
+    <type fullname="System.Runtime.Intrinsics.Vector256">
```

Comment:
Is this true for all codegens (e.g. interpreter)

Reply:
Looks like interpreter mark all vectors as not being hardware accelerated:

```
} else if (in_corlib &&
		   (!strncmp ("System.Runtime.Intrinsics", klass_name_space, 25) &&
			!strncmp ("Vector", klass_name, 6) &&
			!strcmp (tm, "get_IsHardwareAccelerated"))) {
	*op = MINT_LDC_I4_0;
}
```

so for that case it should be ok.

Reply:
cc @kg

Reply:
We mark v128 as hardware accelerated in interp in some cases, I believe.

Reply:
ok, so then this should be OK since it only affects v256/v512.